master/pkg/model/experiment.go

```diff
@@ -531,24 +531,24 @@ func TrialMetricsJSONPath(isValidation bool) string {
 }
 
 // TrialSummaryMetricsJSONPath returns the JSON path to the trials metric summary.
-func TrialSummaryMetricsJSONPath(metricGroup MetricGroup) string {
```

Comment:
why do we change this? we see the legacy metric types as a subset of MetricGroup so I think those should all be able to use the same go type